packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -51,8 +52,8 @@ export class SuggestionsItem<T> extends BaseComponent<ISuggestionItemProps<T>, {
         { this.props.showRemoveButton ? (
           <IconButton
             iconProps={ { iconName: 'Cancel', style: { fontSize: '12px' } } }
-            title='Remove'
-            ariaLabel='Remove'
+            title={ removeButtonAriaLabel }
```

Comment:
>removeButtonAriaLabel [](start = 20, length = 21)

Do we need a default value here? Something like removeButtonAriaLabel || 'Remove;

what happens if removeButtonAriaLabel is undefined ?

Reply:
@manishgarg1 our policy has been no hardcoded strings.